### PR TITLE
JSAE-77:--finished setup aws rds and circleci deployment

### PR DESCRIPTION
**Pull request by amelieykw**

### CODE REVIEW

File: back-end/.env.prod [hunk 1]
```diff
@@ -0,0 +1 @@
+MYSQL_URL=mysql://emerald_admin:password@jsa-emerald-db-instance.cczaaflvgs1d.ap-northeast-1.rds.amazonaws.com:3306/jsa_emerald_db
```
Comment (adamgyulavari) [inline back-end/.env.prod]: @amelieykw whopsie
I'd recommend to change that database password now and than actually remove this env file from git

File: back-end/migration.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-require('dotenv').config()
+require('dotenv').config({ path: `${__dirname}/.env.prod` })
```
Comment (adamgyulavari) [inline back-end/migration.js]: @amelieykw it should be the default `.env` just the `.env` file should have different content

File: back-end/services/dbService.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-require('dotenv').config()
+require('dotenv').config({ path: `${__dirname}/.env.prod` })
```
Comment (adamgyulavari) [inline back-end/services/dbService.js]: @amelieykw same

File: back-end/migration.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-require('dotenv').config()
+require('dotenv').config({ path: `${__dirname}/.env` })
```
Comment (adamgyulavari) [inline back-end/migration.js]: @amelieykw I think that's the default:
```suggestion
require('dotenv').config()
```

Comment (amelieykw) [reply back-end/migration.js]: done

Comment (amelieykw) [reply back-end/migration.js]: @adamgyulavari

**Pull request merged**